README.md

```diff
@@ -98,6 +98,24 @@ It also enable to read tone data from text.
 When you paste the tone data text in "Read text" dialog ("File" -> "Read Text"), this app reads digits from the text and set to tone parameters in the order specified by "Setup" -> "Input".  
 Note: This app reads digits in the order of appearance. This means that it also reads digits included in comments like tone name. Please remove the comments before reading.
 
+## Build instructions
+### Dependecies
+
+The following dependecies are required for building under Ubuntu/Debian Linux:
+`qt5-default`
+`qttools5-dev-tools`
+`qt5-multimedia5-dev`
+`libqt5multimedia5-plugins`
+`libasound2-dev`
+`libpulse-dev //OPTIONAL(?)` 
```

Comment:
Both of these aren't needed for the build, I think. The necessary audio libraries are linked into the Qt5 plugins, not this application.

Reply:
thanks for pointing this out, fixed

Reply:
Technically `libqt5multimedia5-plugins` isn't needed to build the application either, it's a runtime dependency.

E.g. my local package only has these package dependencies:

- `qmake`
- `qtbase`
- `qtmultimedia`

`qtmultimedia` includes the plugins provided by `libqt5multimedia5-plugins` on your system. YM2608 Tone Editor only links against Qt5's Multimedia framework, the plugins are dynamically found & loaded when the applications starts & requests audio access from Qt Multimedia.